dashboard/test/controllers/course_offerings_controller_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class CourseOfferingsControllerTest < ActionController::TestCase
```

Comment:
I would add a test for the `update` function and making sure the correct fields are updated. I'm not sure how useful it is, especially with a simpler data model